src/filedata.h

```diff
@@ -74,7 +70,11 @@ struct FileDataChangeInfo {
 	gboolean regroup_when_finished;
 };
 
-struct FileData {
+class FileData {
```

Comment:
Reminder for the next PR. Convert forward declarations from `struct FileData` to `class FileData`.